api/envoy/extensions/filters/http/bandwidth_limit/v3alpha/bandwidth_limit.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.bandwidth_limit.v3alpha;
+
+import "envoy/config/core/v3/base.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.bandwidth_limit.v3alpha";
+option java_outer_classname = "BandwidthLimitProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Bandwidth limit]
+// Bandwidth limit :ref:`configuration overview <config_http_filters_bandwidth_limit>`.
+// [#extension: envoy.filters.http.bandwidth_limit]
+
+// [#next-free-field: 6]
+message BandwidthLimit {
+  // Defines the mode for the bandwidth limit filter.
+  // Values represent bitmask.
+  enum EnableMode {
+    // Filter is disabled.
+    Disabled = 0;
+
+    // Filter enabled only for incoming traffic.
+    Decode = 1;
+
+    // Filter enabled only for outgoing traffic.
+    Encode = 2;
+
+    // Filter enabled for both incoming and outgoing traffic.
+    DecodeAndEncode = 3;
+  }
```

Comment:
https://developers.google.com/protocol-buffers/docs/style#enums

> Use CamelCase (with an initial capital) for enum type names and CAPITALS_WITH_UNDERSCORES for value names:

Reply:
Ok. Fixing.

Reply:
Done. would it be something that should be caught by the format checks?